Kajabi/sage-lib
# [Sage 493] Quick Fix for subtle button padding

_Note: merging directly into `main` to patch up the 4.18.0 bump_

## Description

With the Sage 3.0 button conversion, `sage-btn--subtle` padding that was introduced for the `hover` effect, which adjusted numerous subtle buttons in the UI to the left by 8px. This task:

* Removes the added `padding` from the [button work](https://github.com/Kajabi/sage-lib/pull/575/files#diff-8b8601a12655a3ec410c2968329becfdd300a5f88fdf21883467af2e7c3d8767R415) 
* Adjusts the `::after` outline mixin in order to hack into the `::after` to accomplish `hover` as designed


## Screenshots

### Issue being corrected
![some of the many alignment issues](https://user-images.githubusercontent.com/24237393/135647584-d74c61f7-e595-46d4-ad5b-29dcadc640ae.png)

### AFTER
![Screen Shot 2021-10-01 at 10 38 14 AM](https://user-images.githubusercontent.com/24237393/135648276-697624e3-1faf-4f28-89d9-220d182c0e7e.png)

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/component/button#element-button
2. Verify the "Subtle Buttons" are no longer padded left and right and the hover/focus states are correct


## Testing in `kajabi-products`
A quick fix which goes into version bump 4.18.0; check for visual regressions in all subtle buttons
